certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -3780,7 +3965,51 @@
           }
         },
         "x-ms-visibility": "important",
-        "summary": "Get Subscription Status",
+        "summary": "Get Subscription Status – Flowr and Vertr",
+        "description": "Get the the status of your Encodian Subscription - https://support.encodian.com/hc/en-gb/articles/360010176717-Get-Subscription-Status"
+      }
+    },
+    "/api/v1/Mgmt/GetTrigrSubscriptionStatus": {
+      "get": {
+        "tags": [
+          "Mgmt"
+        ],
+        "operationId": "GetTrigrSubscriptionStatus",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "schema": { }
```

Comment:
@Encodian can you please explain why you have an empty schema? please define a proper schema.

Reply:
@Amjed-Ayoub - It is empty because there is no value passed in the body. This is exactly the same as the previously approved '/api/v1/Mgmt/GetOperationStatus' action.  I would request this is also approved as it matches previously approved convention.